src/GitHub.TeamFoundation.14/Home/GitHubHomeSection.cs

```diff
@@ -107,6 +147,36 @@ void DoOpenOnGitHub()
             visualStudioBrowser?.OpenUrl(ActiveRepo.CloneUrl.ToRepositoryUrl());
         }
 
+        void ShowWelcomeMessage()
+        {
+            var welcomeMessageGuid = new Guid("C529627F-8AA6-4FDB-82EB-4BFB7DB753C3");
```

Comment:
Probably a good idea to put this and other GUID next to the other GUIDs we have in the Guids class, so they're easy to keep track of